### ISSUE TITLE
Rule proposal: AX_ARIA_12

### ISSUE THREAD

**Issue by lencioni** [evcohen/eslint-plugin-jsx-a11y#14]
I think that this would make a really good linter rule. https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_12

This element does not support ARIA roles, states and properties

Some HTML elements should not be given ARIA attributes. This is often because they are not visible, for example meta, html, script, style.

Bad:

```html
<!-- Bad: the meta element should not be given any ARIA attributes -->
<meta charset="UTF-8" aria-hidden="false">
```

Good:

```html
<!-- Good: the meta element should not be given any ARIA attributes -->
<meta charset="UTF-8">
```
